client/deployment_create.go

```diff
@@ -59,15 +65,16 @@ type DeploymentResponse struct {
 	Build struct {
 		Environment []string `json:"env"`
 	} `json:"build"`
-	AliasAssigned    bool    `json:"aliasAssigned"`
-	ChecksConclusion string  `json:"checksConclusion"`
-	ErrorCode        string  `json:"errorCode"`
-	ErrorMessage     string  `json:"errorMessage"`
-	ID               string  `json:"id"`
-	ProjectID        string  `json:"projectId"`
-	ReadyState       string  `json:"readyState"`
-	Target           *string `json:"target"`
-	URL              string  `json:"url"`
+	AliasAssigned    bool      `json:"aliasAssigned"`
+	ChecksConclusion string    `json:"checksConclusion"`
+	ErrorCode        string    `json:"errorCode"`
+	ErrorMessage     string    `json:"errorMessage"`
+	ID               string    `json:"id"`
+	ProjectID        string    `json:"projectId"`
+	ReadyState       string    `json:"readyState"`
+	Target           *string   `json:"target"`
+	URL              string    `json:"url"`
+	GitSource        GitSource `json:"git_source"`
```

Comment:
mmm, as per their [recommendations](https://www.terraform.io/plugin/sdkv2/best-practices/naming) it looks fine:
> Attribute names within Terraform configuration blocks are conventionally named as all-lowercase with underscores separating words, as shown above.

Reply:
feel free to change it if you think it is more consistent

Reply:
you're right, I just tested it
```
    },
    "gitSource": {
        "repoId": 449314886,
        "ref": "dario-terraform-deployment-test",
        "type": "github",
        "sha": "f4f09e995618dea3cd77e39a2cb710628ae03228"
```
changed

Reply:
Hooray. We should probably set this back onto the terraform state, rather than using the value from `plan` in that case (so it's readable). However... I think we probably need to fix the `repoId` issue with different git providers first.